### PR TITLE
Added MeetingRecordingExpirationDays to New-CsTeamsMeetingPolicy.md

### PR DESCRIPTION
**Pull request by dariomws**

### CODE REVIEW

**Review by get-itips — APPROVED** [review]
Can we add a comment about this?

> The CMD to preemptively change the MeetingExpirationDays setting in Teams (Set-CsTeamsMeetingPolicy -Identity Global -MeetingRecordingExpirationDays) isn't yet available to be set. We'll publish an updated message center post when the setting is available for modification.

https://docs.microsoft.com/en-us/microsoftteams/cloud-recording

**Comment by dariomws** [comment]
> Can we add a comment about this?
> 
> > The CMD to preemptively change the MeetingExpirationDays setting in Teams (Set-CsTeamsMeetingPolicy -Identity Global -MeetingRecordingExpirationDays) isn't yet available to be set. We'll publish an updated message center post when the setting is available for modification.
> 
> https://docs.microsoft.com/en-us/microsoftteams/cloud-recording

Done!

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -941,17 +949,15 @@ Accept wildcard characters: False
 ```
 
 ### -MeetingRecordingExpirationDays
-
 Specifies the number of days before meeting recordings will expire and move to the recycle bin. Value can be from 1 to 99,999 days.
 
-> [!NOTE]
-> You may opt to set Meeting Recordings to never expire by entering the value -1.
-
+Note: This parameter isn't yet available to be set. We'll publish an updated message center post when the setting is available for modification. Please refer to the [roadmap (Feature ID: 84580)](https://www.microsoft.com/microsoft-365/roadmap?searchterms=82057&filters=&searchterms=84580) for more information on its delivery date.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:954]:
```suggestion
Note: This parameter isn't yet available to be set. We'll publish an updated message center post when the setting is available for modification. For more information about its delivery date, see the [roadmap (Feature ID: 84580)](https://www.microsoft.com/microsoft-365/roadmap?searchterms=82057&filters=&searchterms=84580).
```

**Pull request merged**